src/main.ts

```diff
@@ -1,8 +1,10 @@
 import "./app.postcss";
 import { loadTranslations } from "$lib/translations/translations";
 import App from "./App.svelte";
-
-loadTranslations("english");
+import { launcherConfig } from "$lib/config";
+/**Language should be loaded here (point before sidebar, head, ) */
+await launcherConfig.loadConfigFromFile();
+loadTranslations(launcherConfig.language);
```

Comment:
This shouldn't be here, add the `loadConfigFromFile()` to the top of `loadTranslations()` See the other function examples in `lib/config` for more info.